Bourreau/script/cbrain_remote_ctl

```diff
@@ -234,20 +235,25 @@ File.rename(environment_log_file, environment_log_file.sub(/\.log$/,"_#{date_ext
 # Start the server or the console
 raw_system_cmd       = ""
 Dir.chdir(rails_home) do
+  ENV["SECRET_KEY_BASE"] ||= "JUNK-NOT-USED-IN-BOURREAU-THERE-ARE-NO-SESSIONS-ANYWAY"
   rails_cmd_ar = (mode == "start") ?
         [
           "script/rails", "server", "puma",
+          #"puma",
           "-d", # the -d will start it in background
           "-p", "#{port}",
           "-e", "#{environment}",
           "-b", "127.0.0.1",
-          "-P", "#{pidfile}"
+          "-P", "#{pidfile}",
+          #"--pidfile", "#{pidfile}",
+          #"--redirect-stdout", "log/stdout.tmp",
+          #"--redirect-stderr", "log/stderr.tmp",
```

Comment:
Why did you keep comments?

Reply:
There are two ways to launch the server: `rails server puma [options]` and `puma [options]`. Normally the Bourreau is launched with the first way. While I was debugging the GPC problem I tried the second way too, which has slightly different options too. Since in the future maybe we'll want to launch puma directly, I left it there as a form of documentation.

Reply:
Note: the `options` sets are incompatible,